docs/en/sql-reference/table-functions/url.md

```diff
@@ -25,6 +25,10 @@ url(URL, format, structure)
 
 A table with the specified format and structure and with data from the defined `URL`.
 
+**Globs in URL**
+
+Patterns in curly brackets `{ }` are used to generate a set of shards or to specify failover addresses. Supported pattern types and examples see in the description of the [remote](remote.md#globs-in-addresses) function. Additionally, character `|` is used to specify failover addresses. They are iterated in the same order as listed in the pattern. The maximum number of generated addresses is limited by [glob_expansion_max_elements](../../operations/settings/settings.md#settings-glob_expansion_max_elements) setting.
```

Comment:
Not only `Patterns in curly brackets `{ }` are used to ...`, but may be better `Patterns in curly brackets `{ }` and commas are used to ...`, becuase there might be something like `shard1, shard2, shard3` without any `{ }` - case for shards with one replica on each.

Reply:
Are addresses separated with commas without `{ }` supported both by `url()` and `remote()`?
Seems that `url()` does not support this syntax. At least I failed to execute such example...